src/org/parosproxy/paros/extension/option/OptionsViewPanel.java

```diff
@@ -125,10 +134,25 @@ public OptionsViewPanel() {
 	private void initialize() {
         this.setLayout(new CardLayout());
         this.setName(Constant.messages.getString("view.options.title"));
-        this.add(getPanelMisc(), getPanelMisc().getName());
-
+        this.add(getScrollPane(), getPanelMisc().getName());
+	}
+	
+	private JScrollPane getScrollPane() {

```

Comment:
The getter should be named after the variable (in the current case `getViewsScrollPane`).